src/Tau.jl

```diff
@@ -45,9 +45,54 @@ function cisτ(x::Integer)
     return complex(one(y), zero(y))
 end
 
+# Aliases
 const sintau = sinτ
 const costau = cosτ
 const sincostau = sincosτ
 const cistau = cisτ
 
+# Documentation
+
+"""
+    sinτ(x)
+    sintau(x)
+
+Compute ``\\sin(\\tau x)`` more accurately than `sin(tau*x)`, especially for large `x`.
+
+See also: `costau`, `sincostau`.
+"""
+function sintau end
```

Comment:
Note that we cannot find documentation for `sinτ`.
```
help?> sintau
search: sintau sincostau isinteractive

  sinτ(x)
  sintau(x)

  Compute \sin(\tau x) more accurately than sin(tau*x), especially for large x.

  See also: costau, sincostau.

help?> sinτ
"sinτ" can be typed by sin\tau<tab>

search: sinτ sincosτ sinh sind sinc sin sinpi sincos sintau sincosd sincospi sincostau using isinf asinh asind asin isinteger isinteractive thisind sign signed Signed signbit significand islink isfinite occursin Unsigned unsigned missing Missing lastindex MissingException VERSION versioninfo

  No documentation found.

  Tau.sinτ is a Function.
```
I think `sinτ` should be removed (#49), so removing `sinτ` in the docstring would be better.

```suggestion
"""
    sintau(x)

Compute ``\\sin(\\tau x)`` more accurately than `sin(tau*x)`, especially for large `x`.

See also: `costau`, `sincostau`.
"""
function sintau end
```

Reply:
Seems to work as expected with the latest commit (https://github.com/JuliaMath/Tau.jl/pull/47/commits/c0eb0216f8740cadc7ee9e80617e3b12da592879), apparently one has to add the docstring to the "original" function, not its alias.